assets/js/src/newsletters/templates.jsx

```diff
@@ -101,6 +101,9 @@ class NewsletterTemplates extends React.Component {
     const categoriesNames = templatesCategories.map((category) => category.name);
     let categories;
 
+    if (categoriesNames.indexOf('woocommerce') === -1) {
```

Comment:
From the specification, It seems that we should show those templates if WooCommerce is active. This looks like you hide those templates always, don't you?

Reply:
No, I don't. If WooCommerce is active, we do add `woocommerce` category on line 34, which makes this condition false.
In other terms, we will always have the `woocommerce` category but we will show the WC tab only when WC is active. Makes sense?